app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
@chessbyte see @Fryguy's comment lower, I wanted to unify the doctype, because we used 3 different.